### PR TITLE
chore: allow extending .envrc with .envrc.private

### PR DESCRIPTION
**Pull request by busla**
## What

Extend .envrc to allow loading custom .envrc files

## Why
.envrc should allow developers to load their own custom configurations along with the defaults set by team core.

### CODE REVIEW

**Review by peturgq — APPROVED** [review]

**Comment by saevarma** [comment]
I thought that the idea with `.envrc` was to share common unsensitive environment settings recommended for the repo, while programmers can use `.env.secret` to add custom (and sensitive) environment settings that are not committed.

Have you any thoughts on if/how we should share common env settings to developers?

**Review by eirikurn — CHANGES_REQUESTED** [review]
This is likely to change current dx workflows for all developers. E.g. with get-secrets. 

Imo should close this PR and discuss strategies for supporting alternate env workflows on Slack if it's important.

**Comment by busla** [comment]
reopened the PR with a fix.

`direnv` will now load `.envrc.private` if it exists for custom config

File: apps/application-system/api/src/environments/environment.ts
Comment (saevarma) [inline apps/application-system/api/src/environments/environment.ts:1]: Is this file change supposed to be part of this PR

**Comment by saevarma** [comment]
You should update the PR's title to reflect the changes, for the generated changelog.

**Review by saevarma — APPROVED** [review]
LGTM

**Pull request merged**